frescobaldi_app/widgets/linenumberarea.py

```diff
@@ -57,7 +57,7 @@ def sizeHint(self):
     def updateWidth(self):
         fm = QFontMetrics(self._textedit.font())
         text = format(self._textedit.blockCount(), 'd')
-        self._width = fm.width(text) + 3
+        self._width = fm.boundingRect(text).width() + 3
```

Comment:
There are several more calls to `QFontMetrics.width()` in the codebase, but so far I didn't manage to make Frescobaldi execute them and crash, so I didn't fix them, as I hate editing code blindly (and some of them may very well be dead code which should be deleted rather than updated).

Reply:
You are a far more responsible programmer than me; I've been making most of my changes with `sed` :)

According to [here](https://doc.qt.io/qt-5/qfontmetrics-obsolete.html#width) the intended replacement is `QFontMetrics.horizontalAdvance()`, which is what I've used in other spots where this has come up. That said, I haven't worked much with QFontMetrics before. Is there a reason to use `boundingRect().width()` instead?

Reply:
No sound reason, just my insufficient familiarity with Qt documentation. Replaced in 98ae920

Reply:
Fair enough!